src/button/styles/button.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/variables.css';
+
+.button {
+	background-color: var(--button-bg);
+	border: 1px solid var(--button-bg);
+	box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.25);
+	border-radius: 5px;
+	color: var(--button-color);
+	cursor: pointer;
+	font: var(--base-font);
+	font-weight: bold;
+	display: inline-block;
+	padding: 0.75em 1.25em;
+	transition: all 0.3s ease;
+}
+
+.button:hover,
+.button:focus,
+.button[aria-pressed="true"] {
+	background-color: var(--button-hover-bg);
+	border-color: var(--button-hover-bg);
+	outline: none;
+}
+
+.button[disabled],
+.button[disabled]:hover {
+	background-color: var(--button-disabled-bg);
+	border-color: var(--button-disabled-bg);
+	cursor: default;
+}
+
+.button[data-dojo-icon]:before {
+	content: attr(data-dojo-icon);
+	display: inline-block;
+	font-family: var(--button-icon-font);
+	speak: none;
+	font-size: 1.4em;
+	font-style: normal;
+	font-weight: normal;
+	font-variant: normal;
+	text-transform: none;
+	line-height: 1;
+	margin: -0.5em 0.3em -0.5em -0.3em;
+	vertical-align: sub;
+	-webkit-font-smoothing: antialiased;
```

Comment:
Unfortunately in this case, the font smoothing used differs between -webkit and -moz